SimpleMVVM/ViewControllers/ViewController.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  ViewController.swift
+//  SimpleMVVM
+//
+//  Created by Дмитрий Станкевич on 10.02.22.
+//
+
+import Foundation
+import UIKit
+import SnapKit
+
+class ViewController: UIViewController {
+    //MARK: - Varibles
+    private var viewModel = ViewModel()
```

Comment:
In the future please make `viewModel` as optional type and make Initializer for ViewController that requires viewModel